worlds/AutoWorld.py

```diff
@@ -7,8 +7,7 @@
 import time
 from random import Random
 from dataclasses import make_dataclass
-from typing import (Any, Callable, ClassVar, Dict, FrozenSet, Iterable, List, Mapping, Optional, Set, TextIO, Tuple,
-                    TYPE_CHECKING, Type, Union)
+from typing import Any, Callable, ClassVar, FrozenSet, Iterable, Mapping, TextIO, TYPE_CHECKING, Type, Optional
```

Comment:
```suggestion
from typing import Any, Callable, ClassVar, FrozenSet, Iterable, Mapping, TextIO, TYPE_CHECKING
```
There is another usage of `Type` at `options_dataclass` - Github won't let me comment on that line since you didn't change it.